init-service/src/main/java/uk/ac/ox/ndph/mts/init_service/loader/Loader.java

```diff
@@ -29,13 +31,19 @@ public Loader(Trial trialConfig, PractitionerServiceInvoker practitionerServiceI
         this.siteServiceInvoker = siteServiceInvoker;
     }
 
-
     @Override
     public void run(String... args) {
+        // TODO: remove this temporary fix in this story: https://ndph-arts.atlassian.net/browse/ARTS-362
+        try {
+            // Give the other services some time to come online.
+            Thread.sleep(delayStartInSeconds * 1000);
```

Comment:
do we still need this after having the retry mechanism?

Reply:
I thought it's a short way to save some retries and errors in the log. And anyway, we can remove it if/when we're confidant Terraform brings this service up at the appropriate time (hence the TODO comment)